docs/EN_US/RuningHPCCinAmazonWebServicesEC2/RuningHPCCinAmazonWebServicesEC2.xml

```diff
@@ -289,15 +289,15 @@
                     <para><emphasis role="bold">TCP ports 1-65535</emphasis>
                     (for internal access only)</para>
 
-                    <para>Type the Group ID for the HPCC group in the Source
```

Comment:
I'm not sure that HPCC in this context refers to HPCC Systems.  Should it be hpcc group?  What exactly does it correspond to?